### PR TITLE
Making ParseError and ValidationError public

### PR DESCRIPTION
**Pull request by msdrigg**
- Solves #101

### CODE REVIEW

**Comment by msdrigg** [comment]
I tested this in my own repo and it's working.

**Comment by fmeringdal** [comment]
LGTM! Make sure to add an entry in `CHANGELOG.md` that these errors are made public and then this should be good to merge

**Comment by msdrigg** [comment]
Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## 0.13.0 (2024-02-05)
+
+- Make `ParseError` and `ValidationError` public
```
Comment (fmeringdal) [inline CHANGELOG.md:10]: This needs to be added to the `0.12.0` entry as that is the next version to be released

**Review by fmeringdal — CHANGES_REQUESTED** [review]

**Pull request merged**